config/jobs/kubernetes/sig-network/sig-network-kind.yaml

```diff
@@ -321,7 +319,7 @@ periodics:
     preset-kind-volume-mounts: "true"
   decorate: true
   decoration_config:
-    timeout: 200m
```

Comment:
Did you mean to change the timeouts?

Reply:
yes, some jobs were serial and I prefer to run the parallel , because they add more stress

Reply:
see 
```
    - name: "PARALLEL"
        value: "true"
```

is added on those